salt/docker/stop.sls

```diff
@@ -0,0 +1,4 @@
+# Stop and disable the docker daemon
+docker:
+  service.dead:
+    - enable: False
```

Comment:
As this is using systemd, and I'm not 100% certain what  the issue is + I'm not seeing any issues.. I've left this out for now.